README.md

```diff
@@ -471,12 +471,14 @@ To run the new tests, you will need to first install karma globally:
 yarn global add karma
 ```
 
-You can run the new tests with the command
+You can run tests with the command
 
 ```bash
 karma start
 ```
 
+If called in the top level ./ directory, the unit tests will run. If called
```

Comment:
This is a bit confusing.. can we change it so it's controlled by a flag and should always be run from the root folder?

Reply:
I found a way to run from root with karma. Also, if you run the grunt command from root, it runs the integration tests and the unit tests by default. Updated the README with this information.